lib/translateobjects.js

```diff
@@ -0,0 +1,35 @@
+var _ = require('lodash');
+var dateMetaFieldName = "$date";
+
+function translateObjects(fields) {
+    var ret = fields;
+    if (fields) {
+        _.forOwn(fields, function(val,key,obj) {
```

Comment:
This will only check one level deep in the collections. Should we consider recursively checking all levels?

Reply:
Could do, not really sure how many people would use nested Date objects and DataBrowser.  The more correct way would be to recurse, would probably have to add a limit also though.  Any idea of how deep any current production values would go to?

Reply:
Agreed, it is not worth the change